src/components/Alerts/RemediationStatusToast.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import propTypes from 'prop-types';
+
+import { Alert } from '@patternfly/react-core';
+import './RemediationStatusToast.scss';
+
+function RemediationStatusToast ({ status, name, ...props }) {
```

Comment:
Shouldn't we use for the Toast notifications use FEC notifications?

Reply:
Yeah we definitely can go down that route. If that's easier, I can remove the toast from this PR.